### PR TITLE
[one-cmd] Add section, input_path, output_path for one-init

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
This commit enables one-init to add sections and
input_path and output_path of each section.

for #9450
draft #9514

ONE-DCO-1.0-Signed-off-by: Hyun Sik Yoon <hyunsik.yoon.1024@gmail.com>

### CODE REVIEW

File: compiler/one-cmds/one-init [hunk 1]
```diff
@@ -275,6 +365,11 @@ def main():
     # verify arguments
     _verify_arg(parser, args)
 
+    #TODO generate cfg file
```
Comment (yunjayh) [inline compiler/one-cmds/one-init:368]: AFAIU, `_generate()` writes cfg file. What does `TODO` in the comment mean?

Comment (hyunsik-yoon) [reply compiler/one-cmds/one-init]: Thanks. I removed!

File: compiler/one-cmds/one-init [hunk 2]
```diff
@@ -235,16 +284,40 @@ def _generate():
         pass  # NYI
 
     def _gen_import():
-        pass  # NYI
+        section = f'one-import-{model_type}'
```
Comment (seanshpark) [inline compiler/one-cmds/one-init:287]: Q) where is `model_type` from?

Comment (seanshpark) [reply compiler/one-cmds/one-init:287]: Ah, `_gen_import` is inner function of `_generate`... I'm not good with python and learned now about this :)

File: compiler/one-cmds/one-init [hunk 3]
```diff
@@ -264,6 +337,23 @@ def _generate():
...
+        _, ext = os.path.splitext(args.input_path)
+
+        # ext would be, e.g., '.tflite' or '.onnx'.
+        # Note: when args.input_path does not have an extension, e.g., '/home/foo'
+        # ext after os.path.splitext() is '' and ''[1:] is still ''.
+        ext = ext[1:]
+        if ext in ["tflite", "onnx"]:
+            return ext
+
+    parser.error(f'the following argument is required: --input_path')
```
Comment (seanshpark) [inline compiler/one-cmds/one-init:354]: `f'...'` is used but seems no variable used for format.

Comment (seanshpark) [reply compiler/one-cmds/one-init:354]: if `ext` is not one of `tflite` nor `onnx` this error will be shown even if `--input_path` is given, which seems logical error.

**Comment by yunjayh** [comment]
Substituted to #9813.

**Pull request closed without merging**